### PR TITLE
Lk methylome release

### PR DESCRIPTION
**Pull request by ekiernan**
### Description
Moved methylome to pipelines folder, added documentation, and a changelog
----

### Checklist 
If you can answer "yes" to the following items, please add a checkmark next to the appropriate checklist item(s) **and** notify our WARP documentation team by tagging either @ekiernan or @kayleemathews in a comment on this PR.

- [ ] Did you add inputs, outputs, or tasks to a workflow?
- [ ] Did you modify, delete or move: file paths, file names, input names, output names, or task names?
- [ ] If you made a changelog update, did you update the pipeline version number?

### CODE REVIEW

File: .dockstore.yml [hunk 1]
```diff
@@ -7,6 +7,10 @@ workflows:
     testParameterFiles:
           -  /warp/pipelines/skylab/optimus/example_inputs/human_v3_example.json
 
+  - name: snM3C
```
Comment (nikellepetrillo) [inline .dockstore.yml:10]: we already have snM3C in the dockstore yml (around line 114) do you mind just removing those lines? 114-116

File: website/docs/Pipelines/snM3C/README.md [hunk 2]
```diff
@@ -0,0 +1,92 @@
+---
+sidebar_position: 1
```
Comment (nikellepetrillo) [inline website/docs/Pipelines/snM3C/README.md:2]: i can see this when i "display the rich diff", im not sure if we want to be able to see this?

File: website/docs/Pipelines/snM3C/README.md [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+
+The snM3C workflow is a cloud-based computational workflow for processing single-nucleus methylome and chromatin contact (snM3C) sequencing data. The workflow is designed to demultiplex raw sequencing reads, align them, call chromatin contacts, and generate summary metrics. It is developed in collaboration with the laboratory of Hanqing Liu and Joseph Ecker. For more information about the workflow, please see the [YAP documentation](https://hq-1.gitbook.io/mc/).
+
+## Set-up
+
+### Installation
+
+To use the latest release of the snM3C pipeline, visit the [WARP releases page](https://github.com/broadinstitute/warp/releases) and download the desired version.
+
+<!--- Add a comment about running an old version of the workflow --->
```
Comment (jessicaway) [inline website/docs/Pipelines/snM3C/README.md:22]: I'm assuming this is a comment for future us and won't render in the meantime, is that right?

File: website/docs/Pipelines/snM3C/README.md (hunk 2 above)
Comment (ekiernan) [reply website/docs/Pipelines/snM3C/README.md:2]: It won't render on the doc site. This should be ok.

File: website/docs/Pipelines/snM3C/README.md (hunk 3 above)
Comment (ekiernan) [reply website/docs/Pipelines/snM3C/README.md:22]: Nope, it won't render on the doc site. It should be ok.

File: .dockstore.yml (hunk 1 above)
Comment (ekiernan) [reply .dockstore.yml:10]: Thank you! Just removed.

File: website/docs/Pipelines/snM3C/README.md [hunk 4]
```diff
@@ -0,0 +1,92 @@
+---
+sidebar_position: 1
+slug: /Pipelines/snM3C/README
+---
+# snM3C Overview
+
+| Pipeline Version | Date Updated | Documentation Authors | Questions or Feedback |
+| :----: | :---: | :----: | :--------------: |
+| [snM3C_v1.0.0](https://github.com/yourusername/snM3C/releases) | August, 2023 | [Kaylee Mathews](mailto:warp-pipelines-help@broadinsitute.org) | Please file GitHub issues in the [WARP repository](https://github.com/broadinstitute/warp/issues) |
```
Comment (nikellepetrillo) [inline website/docs/Pipelines/snM3C/README.md:9]: this link https://github.com/yourusername/snM3C/releases doesnt work for me... is the yourusername supposed to auto fill in?

Comment (ekiernan) [reply website/docs/Pipelines/snM3C/README.md]: I wish! No, that's just incorrect. Thank you for catching.

**Review by jessicaway — APPROVED** [review]
Looks good!

**Review by nikellepetrillo — APPROVED** [review]
thanks liz!

**Pull request merged**